### PR TITLE
2nd batch of changes based on B->pi form factor project with Blazenka and Domagoj

### PR DESCRIPTION
**Pull request by dvandyk**

### CODE REVIEW

**Comment by DomLeljak** [comment]
line 1943

- `const double b = this->m_b_msbar(mu);`

in `eos/form-factors/analytic-b-to-pi.cc` has probably been removed before.

File: eos/form-factors/analytic-b-to-pi_TEST.cc [hunk 1]
```diff
@@ -248,22 +248,62 @@ class AnalyticFormFactorBToPiDKMMO2008Test :
...
+                TEST_CHECK_NEARLY_EQUAL( 0.2171, ff_no_rescale.f_t( -5.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.2638, ff_no_rescale.f_t( -1.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.2781, ff_no_rescale.f_t(  0.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.2937, ff_no_rescale.f_t(  1.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.3728, ff_no_rescale.f_t(  5.0), 1. * eps);
+                TEST_CHECK_NEARLY_EQUAL( 0.5326, ff_no_rescale.f_t( 10.0), 1. * eps);
+            }
 
+            {
+                // Comparison with Blazenka's notebook underlysing the [DKKMO:2008A] results
```
Comment (DomLeljak) [inline eos/form-factors/analytic-b-to-pi_TEST.cc:269]: Since you opened a scope here, I guess `Parameters p = Parameters::Defaults();` is missing.

Comment (dvandyk) [reply eos/form-factors/analytic-b-to-pi_TEST.cc:269]: Fixed. Moreover, the actuall form factor object itself was missing. Lazy conflict resolution on my part, I assume.

**Comment by dvandyk** [comment]
macOS builds are failing at the moment, and everything else works fine. Merging!

**Pull request merged**